pkg/apis/core/types_shoot.go

```diff
@@ -691,8 +691,23 @@ type KubeSchedulerConfig struct {
 	// Note that using this field is considered alpha-/experimental-level and is on your own risk. You should be aware
 	// of all the side-effects and consequences when changing it.
 	KubeMaxPDVols *string
+	// Profile configures the scheduling profile for the cluster.
+	// If not specified, the used profile is "balanced" (provides the default kube-scheduler behavior).
+	Profile *SchedulingProfile
 }
 
+// SchedulingProfile is a string alias used for scheduling profile values.
+type SchedulingProfile string
+
+const (
+	// SchedulingProfileBalanced is a scheduling profile that attempts to spread Pods evenly across Nodes
+	// to obtain a more balanced resource usage. This profile provides the default kube-scheduler behavior.
+	SchedulingProfileBalanced SchedulingProfile = "balanced"
+	// SchedulingProfileBinPacking is a scheduling profile that scores Nodes with a high allocation of resources
+	// higher than the others, resulting in a greater overall resource utilizaton.
```

Comment:
```suggestion
	// SchedulingProfileBinPacking is a scheduling profile that scores Nodes with an allocation of resources
	// higher than the others, resulting in a greater overall resource utilization.
```

(same for other API type files)

Reply:
I guess changing "that scores Nodes with a high allocation of resources higher than the others" to "that scores Nodes with an allocation of resources higher than the others" does not make it clear that this profile scores the most busy Nodes higher, does it?

Reply:
Why not? 😄 Aren't "Nodes with an allocation of resources higher than the others" the most busy nodes?

Reply:
Hm, we rather way to say that "the profile scores Nodes (that match something -> have a high allocation of resources) higher than the others (the other Nodes that don't have high allocation of resources).". "higher than the others" part does not relate to the "allocation" but to "scoring".

Reply:
Maybe it's a personal problem, but I don't understand this. Maybe a complete other description (focused more on the actual effect than on such details) is more user-friendly?

Reply:
I changed to
```golang
	// SchedulingProfileBinPacking is a scheduling profile that scores Nodes based on the allocation of resources.
	// It prioritizes Nodes with most allocated resources. This leads the Node count in the cluster to be minimized and
	// the Node resource utilization to be increased.
```

I hope that it is better now.